### PR TITLE
[RTC-389] Add simple docs about DNS distribution strategy

### PR DESCRIPTION
**Pull request by Rados13**
Related to: https://github.com/jellyfish-dev/jellyfish/pull/109

### CODE REVIEW

File: docs/cluster.md [hunk 1]
```diff
@@ -15,20 +15,25 @@ Currently, Jellyfish doesn't offer an option to split a room across multiple mac
 
 ## Configuring a Cluster
```
Comment (mickel8) [inline docs/cluster.md:16]: Let's arrange this whole section in the following way:
1. Strategies (here we need to explain the difference between NODES_LIST and DNS strategies)
1. Configuring a Cluster
    1. NODES_LIST
    1. DNS
1. Examples
    1. NODES_LIST
        1. Running from source
        1. Running with Docker (locally)
        1. Running with Docker (globally)
     1. DNS
 1. Verifying that a cluster has been created
 1. Deeper dive into Erlang Distribution

File: docs/getting_started/installation.md [hunk 2]
```diff
@@ -206,6 +201,22 @@ Only available when running with Docker or `mix release`.
...
+Defaults to `""`.<br/>
+Example: `JF_DIST_NODES="jellyfish1@127.0.0.1 jellyfish2@127.0.0.1"`.<br/>
+This list can also include ourselves so that you can pass the same value
+to every Jellyfish.
+Note: Jellyfish connection to other Jellyfish is transitive meaning that
+when you connect to one Jellyfish you also connect to all other Jellyfishes
+this one was connected to.
+
+###### Distribution DNS specific:
+* `JF_DIST_QUERY` - query which is sent do DNS. <br/>
```
Comment (mickel8) [inline docs/getting_started/installation.md:215]:
```suggestion
* `JF_DIST_QUERY` - query sent to DNS to discover other Jellyfishes. <br/>
```

File: docs/getting_started/installation.md [hunk 3]
```diff
@@ -206,6 +201,22 @@ Only available when running with Docker or `mix release`.
...
+Returned list of IPs from DNS is used for creating distribution node name in the format `<NODE_BASENAME>@<IP_ADDRESS>`.
```
Comment (mickel8) [inline docs/getting_started/installation.md:216]: I belive we should no longer refer to `NODE_BASENAME` as we removed this env. We need to also mention (probably in JF_DIST_NODE_NAME?) that in the case of DNS distribution, every JF has to have the same `nodename` part of `nodename@hostname`.

File: docs/cluster.md [hunk 4]
```diff
@@ -15,20 +15,25 @@ Currently, Jellyfish doesn't offer an option to split a room across multiple mac
...
 
 List of all cluster-related environment variables is available [here](./getting_started/installation#distribution).
 
 :::
 
+:::tip Configure HTTP and metrics ports so they don't overlap with other nodes.
+
+You can do this with `JF_PORT` and `JF_METRICS_PORT` environment variables.
+
+:::
```
Comment (mickel8) [inline docs/cluster.md:35]: This is more related to specific examples and we already mention this in `Running from source` for NODES_LIST strategy and have correct docker-compose examples so I would remove this?

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: docs/getting_started/installation.md (hunk 3 above)
Comment (Rados13) [reply docs/getting_started/installation.md:216]: I mentioned that [here](https://github.com/jellyfish-dev/jellyfish-docs/blob/06dcde196f461be3953dfd4efb10c5b91c94cfc2/docs/cluster.md?plain=1#L79-L80) but if you want I can also mention it with `JF_DIST_NODE_NAME`.

Comment (mickel8) [reply docs/getting_started/installation.md]: Please do

**Review by mickel8 — APPROVED** [review]

**Pull request merged**